src/compose.js

```diff
@@ -7,12 +7,13 @@ const assign = Object.assign;
 
 /**
  * Creates new factory instance.
- * @param {object} descriptor The information about the object the factory will be creating.
+ * @param {Descriptor} descriptor The information about the object the factory will be creating.
  * @returns {Function} The new factory function.
  */
 function createFactory(descriptor) {
   return function Stamp(options, ...args) {
-    const obj = Object.create(descriptor.methods || {});
+    // Next line was optimized for most JS VMs. Please, be careful here!
+    const obj = Object.create(descriptor.methods || null);
```

Comment:
I am wondering, have you actually tried to avoid using `Object.create` itself if methods are not set? You could do something like `descriptors.methods ? Object.create(descriptor.methods) : {};`. I do remember from some benchmarks I was running like 2-3 years ago that this function itself was terribly slow. Also I don't actually see amy point in executing it if you just want a new object